LuLu/Extension/Rules.m

```diff
@@ -319,15 +316,34 @@ -(BOOL)generateDefaultRules
     SecCSFlags flags = kSecCSDefaultFlags | kSecCSCheckNestedCode | kSecCSDoNotValidateResources | kSecCSCheckAllArchitectures;
     
     //dbg msg
-    os_log_info(logHandle, "generating default rules");
-    
+    os_log_debug(logHandle, "generating default rules");
+
     //connects to LuLu Server if server mode is enabled:
-    if (connectToServer)
+    if (LULU_SERVER_MODE)
     {
         //connect to LuLu Server and get the default rules there:
-        NSDictionary* response = [self getDefaultRules];
+         NSDictionary* response = [self getDefaultRules];
     
-        os_log_info(logHandle,"response: %{public}@", response);
+        os_log_debug(logHandle,"response: %{public}@", response);
+
+        if (response != nil) {
+            //iterating over each key in response: 
+            for (NSString* key in response){
+                for (NSDictionary* serverRule in response[key]) {
```

Comment:
esse n2 aqui soa estranho. procurar como acessar os valores de um NSDictionary.

Reply:
// Retrieving all values
NSArray *allValues = [dictionary allValues];

// Printing values
for (id value in allValues) {
    NSLog(@"%@", value);
}